holoviews/core/operation.py

```diff
@@ -178,6 +178,9 @@ def process_element(self, element, key, **params):
         The process_element method allows a single element to be
         operated on given an externally supplied key.
         """
+        if self._per_element and not isinstance(element, Element):
+            return element.clone({k: self.process_element(el, key, **params)
+                                  for k, el in element.items()})
```

Comment:
To clarify, does this mean `_per_element` simply wasn't respected for `DynamicMaps`?